src/main/java/com/google/sps/servlets/ImagesServlet.java

```diff
@@ -1,35 +1,84 @@
-package main.java.com.google.sps.servlets;
+package com.google.sps.servlets;
 
+//getting the images.java class...
+import com.google.sps.data.Images;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.api.client.util.Data;
 import com.google.api.gax.paging.Page;
 import com.google.cloud.storage.Blob;
 import com.google.cloud.storage.Bucket;
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.StorageOptions;
+import com.google.gson.Gson;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
 
 
 @WebServlet("/images")
 public class ImagesServlet extends HttpServlet {
 
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException 
+    {
         String projectId = "spring21-sps-2";
         String bucketName = "spring21-sps-2";
         Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
         Bucket bucket = storage.get(bucketName);
         Page<Blob> blobs = bucket.list();
         
 
-
-        response.setContentType("text/html;");
         for (Blob blob : blobs.iterateAll()){
```

Comment:
So, I think this code can be removed now. Since we're returning the image URL in the JSON, we can just call this servlet and create an image element on the webpage with the `src` attrubute set to the URL from the JSON.

`<img src="https://the-url.from.the/json" />`

@enkere87 Agree?

Reply:
I agree, it would make it easier for the webpage styling as well. I did this in my portfolio.